content/articles/react-components-instantiation/index.md

```diff
@@ -0,0 +1,197 @@
+### React Components  Instantiation
+
```

Comment:
Kindly fix all these spacing between the header and first sentence. Remove them!

Reply:
I have made the corrections.